README.md

```diff
@@ -1,4 +1,13 @@
 Docker Toolbox
+
+### TL/DR: Docker will be ceasing Support and Development of the Docker Toolbox project in 2020 - going forward please use [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+Docker released the Docker Toolbox project to make it easier for developers who work on Mac and Windows to get started using Docker. In 2016 Docker released Docker Desktop which superseded toolbox and was significantly easier for the majority of users to get started.  This still left some users behind, predominantly users who were on Windows Home editions, Windows 7, Windows 8 and users of Virtualbox. 
+Since 2016 there have been a number of changes. Windows 7 is no longer supported and the mainstream support of Windows 8.1 has ended. The majority of Windows users now on a version of Windows 10.  Since [Virtualbox 6.0](https://docs.oracle.com/en/virtualization/virtualbox/6.0/admin/hyperv-support.html#:~:text=Oracle%20VM%20VirtualBox%20can%20be,engine%20for%20the%20host%20system) users have been able to run virtualbox and HyperV at the same time on their Windows machines, allowing users to use virtualbox and Docker Desktop side by side on HyperV. For Windows Home users, WSL 2 is available and Docker Desktop now uses this to provide [Desktop for Windows Home](https://docs.docker.com/docker-for-windows/install-windows-home/)
```

Comment:
typo: missing `are` or `run` or … before `now on a version of Windows 10`?

Reply:
Fixed